ntbs-service/Pages/Notifications/Denotify.cshtml.cs

```diff
@@ -0,0 +1,74 @@
+using Microsoft.AspNetCore.Mvc;
+using ntbs_service.Models;
+using ntbs_service.Models.Enums;
+using ntbs_service.Pages_Notifications;
+using ntbs_service.Services;
+using System;
+using System.Threading.Tasks;
+
+namespace ntbs_service.Pages.Notifications
+{
+    public class DenotifyModel : NotificationModelBase
+    {
+        public ValidationService  ValidationService { get; set; }
```

Comment:
private?

Reply:
Can't be private as need access to it in the view for purposes of `!Model.ValidationService.IsValid("PropertyKey")`

Reply:
oh yeah, that makes total sense 😣

Reply:
oh yeah, that makes total sense 😣